pkg/process/net/common.go

```diff
@@ -81,12 +83,23 @@ func GetRemoteSystemProbeUtil() (*RemoteSysProbeUtil, error) {
 }
 
 // GetProcStats returns a set of process stats by querying system-probe
-func (r *RemoteSysProbeUtil) GetProcStats() (*model.ProcStatsWithPermByPID, error) {
-	req, err := http.NewRequest("GET", fmt.Sprintf("%s", procStatsURL), nil)
+func (r *RemoteSysProbeUtil) GetProcStats(pids []int32) (*model.ProcStatsWithPermByPID, error) {
+	procReq := &model.ProcessRequest{
+		Pids: pids,
+	}
+
+	reqBody, err := reqEncoding.GetMarshaler(reqEncoding.ContentTypeProtobuf).Marshal(procReq)
 	if err != nil {
 		return nil, err
 	}
+
+	req, err := http.NewRequest("POST", fmt.Sprintf("%s", procStatsURL), bytes.NewReader(reqBody))
```

Comment:
nit: sorry this is probably the same with original code, but I think we don't need the `fmt.Sprintf` part.